feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-base.component.spec.ts

```diff
@@ -182,5 +182,37 @@ describe('ConfigUIKeyGeneratorService', () => {
       const techLabel = label + ' / [' + techName + ']';
       expect(classUnderTest.getLabel(true, label, techName)).toEqual(techLabel);
     });
+    it('should return label without technical name but with positive price', () => {
```

Comment:
Plz, remove `without technical name but`. It is irrelevant for your purposes.

Reply:
done